cmd/neofs-oauthz/config.go

```diff
@@ -13,13 +13,14 @@ import (
 )
 
 const (
-	defaultEmailAttr        = "Email"
-	defaultBearerCookieName = "Bearer"
-	defaultBearerLifetime   = 30
-	defaultMaxObjectSize    = 200 << 20 // 200MB
-	defaultConnectTimeout   = 30 * time.Second
-	defaultRebalanceTimer   = 15 * time.Second
-	defaultRequestTimeout   = 15 * time.Second
+	defaultEmailAttr         = "Email"
+	defaultBearerCookieName  = "Bearer"
+	defaultBearerLifetime    = 30
```

Comment:
30 of what? Epochs? Is it described anywhere?

Reply:
Epochs, added the comment for this